src/index_enable_billing_activation.js

```diff
@@ -1,55 +1,106 @@
-var req = require("request");
-var registerAccount = function(accId, params) {
-  var datetime = new Date();
-  var bodyjson = {
+'use strict';
+
+const aws = require('aws-sdk');
+
+const req = require('request');
+
+const fs = require('fs');
+
+const alertMailTemplate = './template/alertmail.txt';
+
+//----------------------------------------------------------------------
+function sendAlertMailBySes(params, errorMsg, billingData) {
+  const ses = new aws.SES();
+  const emailParams = {
+    Destination: {
+      ToAddresses: billingData.billingToEmail.split(',')
+    },
+    Source: billingData.billingFromEmail,
+    Message: {
+      Subject: {
+        Data: 'MCAWS: Account Registration Failed with Billing System'
+      },
+      Body: { Html: { Data: '' } }
+    }
+  };
+  fs.readFile(alertMailTemplate, (err, data) => {
+    if (err) throw err;
+    let maildata = data.toString();
+    // replacing predefine mail template with actual error message
+    maildata = maildata.replace(/__ERROR__/gm, JSON.stringify(errorMsg));
+    maildata = maildata.replace(/__PARAMS__/gm, JSON.stringify(params));
+    emailParams.Message.Body.Html.Data = maildata;
+    ses.sendEmail(emailParams, (error, res) => {
+      if (error) console.log(error, error.stack);
+      else console.log(res);
+    });
+  });
+}
+//----------------------------------------------------------------------
+function registerAccount(accId, params) {
+  const datetime = new Date();
+  const bodyjson = {
     sgId: params.SGID,
     offeringNum: params.OfferingNum,
     masterAwsId: params.masterAwsId,
     customerAwsId: accId,
     customerAwsName: params.name,
-    type: params.type ,
+    type: params.type,
     awsDesc: params.desc,
     activationDate: datetime
-  }
+  };
+  const billingUrl = `${params.billingUrl}?client_id=${params.apiKey}&client_secret=${params.secretKey}`;
+  console.log(billingUrl);
```

Comment:
We should not print or log the access and secret key. Please remove access and secret key from console log.